gym_bridges/envs/test_bridges_env.py

```diff
@@ -9,6 +9,16 @@
 from gym_bridges.envs.bridges_env import BridgesEnv
 
 
+@pytest.mark.parametrize("width", [3, 9])
+def test_force_standard_config_render(width):
+    """The standard configuration only has ground at the bottom left
+    and right locations of the env."""
+    env = BridgesEnv(width=width, force_standard_config=True)
+    env.reset()
```

Comment:
This test doesn't run correctly without `env.reset()`. Is it expected that every `BridgesEnv` instance will be reset before being renderer?

Reply:
I think so... reset() before doing anything else seems to be the convention per the small code snippet here and a few other places I've seen.

https://gym.openai.com/